modules/storages/app/components/storages/project_storages/members/row_component.rb

```diff
@@ -53,7 +53,7 @@ def status
       connection_result = storage_connection_status
 
       if connection_result == :not_connected
-        render(Primer::Beta::Octicon.new(:'alert-fill', size: :small, color: :severe)) +
+        helpers.op_icon('icon-warning -warning') +
```

Comment:
I was writing up this suggestion and then saw a Refresh warning. Nice one! 👍

Reply:
Sweet, thanks! 🏆